filer/migrations/0006_auto_20160623_1627.py

```diff
@@ -16,6 +13,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='image',
             name='file_ptr',
-            field=models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, related_name='%(app_label)s_%(class)s_file', serialize=False, to='filer.File'),
+            field=models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, related_name='%(app_label)s_%(class)s_file', serialize=False, to='filer.File', parent_link=True),
```

Comment:
@yakky same here, do we need to apply it to other migrations as well?

Reply:
wherever `file_ptr` is present in the migration we need `parent_link=True`

See also `tests/utils/custom_image/migrations/0002_auto_20160621_1510.py`